### PR TITLE
CW Additional Planes (LargeCAS and LargeAA)

### PR DESCRIPTION
**Pull request by stutpip123**
trying to add vehiclesPlaneLargeAA and vehiclesPlaneLargeCAS, with intention to be used in all places normal equivalents used, except spawning in hangars

## What type of PR is this?
1. [ ] Bug
2. [x] Change
3. [x] Enhancement
4. [x] Miscellaneous

### What have you changed and why?
Information:
adding new LargeCAS and LargeAA plane categories to facilitate use of planes too large to fit into hangars
CW modset makes use of these new categories

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [x] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
needs thorough testing
********************************************************
Notes:

### CODE REVIEW

**Comment by wersal454** [comment]
Don't forget, carpetbombs and airstrike supports also use CAS planes

**Comment by wersal454** [comment]
> Don't forget, carpetbombs and airstrike supports also use CAS planes

carpetbombs to my understanding are not working at all + if I'm reading code correctly, airstrikes are identical to carpetbombs in terms of what they do. So maybe we could cut them out actually

**Comment by stutpip123** [comment]
changing the airstrike function similar to the others doesnt seem to work, the way it chooses is slightly different

airstike
```private _planeType = selectRandom (Faction(_side) get "vehiclesPlanesCAS");```
compared to
cas + casdive
```private _vehType = selectRandom (_faction get "vehiclesPlanesCAS");```

**Comment by wersal454** [comment]
> changing the airstrike function similar to the others doesnt seem to work, the way it chooses is slightly different
> 
> airstike `private _planeType = selectRandom (Faction(_side) get "vehiclesPlanesCAS");` compared to cas + casdive `private _vehType = selectRandom (_faction get "vehiclesPlanesCAS");`

Try changing ```private _planeType = selectRandom (Faction(_side) get "vehiclesPlanesCAS");``` to ```private _planeType = selectRandom (_faction get "vehiclesPlanesCAS");``` and if it works, simply add new largeplanes

**Comment by stutpip123** [comment]
does not seem to work

**Comment by wersal454** [comment]
> does not seem to work

I'll look into this today, probably

**Comment by wersal454** [comment]
```
private _faction = Faction(_side);
private _planeType = selectRandom ((_faction get "vehiclesPlanesCAS") + (_faction get "vehiclesPlanesLargeCAS"));
``` 
or
 ```
private _planeType = selectRandom (Faction(_side) get "vehiclesPlanesCAS") + (Faction(_side) get "vehiclesPlanesLargeCAS"));```

File: A3A/addons/core/functions/Garrison/fn_checkVehicleType.sqf [hunk 1]
```diff
@@ -50,6 +50,10 @@ params ["_vehicle", "_preference"];
     || {_vehicle in FactionGet(inv,"vehiclesPlanesCAS")} \
     || {_vehicle in FactionGet(inv,"vehiclesPlanesAA")} \
     || {_vehicle in FactionGet(occ,"vehiclesPlanesGunship")} \
+	|| {_vehicle in FactionGet(occ,"vehiclesPlanesLargeCAS")} \
+	|| {_vehicle in FactionGet(inv,"vehiclesPlanesLargeAA")} \
+	|| {_vehicle in FactionGet(occ,"vehiclesPlanesLargeAA")} \
+	|| {_vehicle in FactionGet(inv,"vehiclesPlanesLargeCAS")} \
```
Comment (Silence-112) [inline A3A/addons/core/functions/Garrison/fn_checkVehicleType.sqf:56]: Any reason to do this twice?

Comment (stutpip123) [reply A3A/addons/core/functions/Garrison/fn_checkVehicleType.sqf:56]: one is Occ, one is Inv

**Pull request merged**